lib/chef/provisioning/convergence_strategy/ignore_convergence_failure.rb

```diff
@@ -0,0 +1,53 @@
+class Chef
+module Provisioning
+  class ConvergenceStrategy
+
+    # The purpose of this class is to decore the `converge` method with logic to catch any
+    # convergence failure exceptions, log them and then squelch them.  The reason we
+    # need this is to prevent 1 provisioned node's converge failing an entire provisioning
+    # recipe.
+    module IgnoreConvergenceFailure
+
+      attr_accessor :ignore_failures_array, :ignore_exit_values
+
+      # This module is only meant to be extended into instances, not classes or modules
+      def self.extended(instance)
+        opts = instance.convergence_options[:ignore_failure]
+        instance.ignore_failures_array = []
+        instance.ignore_exit_values = []
+        if opts == true
+          instance.ignore_failures_array << RuntimeError
+        else
+          # We assume it is integers or errors
+          opts = [opts].flatten
+          opts.each do |o|
+            case
+            when o.is_a?(Fixnum)
+              instance.ignore_exit_values << o
+            when o.is_a?(Range)
```

Comment:
I documented it in the [`example in the README`](https://github.com/chef/chef-provisioning/blob/9af761a5d45a4ea8d389df3a3dc5eda3f775edde/README.md#machine-options) - do you think it needs expanded documentation?